source/deployment/guest-accounts.rst

```diff
@@ -1,14 +1,11 @@
 .. _guest-accounts:
 
 Guest Accounts (Beta) (E10/E20)
-===================================
+================================
 
 Available in `Enterprise Edition E10 and higher <https://about.mattermost.com/pricing/>`__.
 
-Guest Accounts are a new role within Mattermost Enterprise editions which allow organizations to collaborate with individuals (such as vendors and contractors) outside of their organization.
-The Guest Account feature controls access to channels and other members within the organization.
-
-For example, an organization may use Guest Accounts to work on a website project with resources from an external design firm. Guest Accounts could also be used to collaborate with customers on a support issue.
+The Guest Account feature provides organizations with a way to collaborate with individuals (such as vendors and contractors) outside of their organization by controlling their access to channels and team members. For example, Guest Accounts can be used to collaborate with customers on a support issue or work on a website project with resources from an external design firm.
```

Comment:
```suggestion
The Guest Account feature provides organizations a way to collaborate with individuals (such as vendors and contractors) outside of their organization by controlling their access to channels and team members. For example, Guest Accounts can be used to collaborate with customers on a support issue or work on a website project with resources from an external design firm.
```

Reply:
How about "Guest Accounts are a way to collaborate..."?

Reply:
@justinegeffen I like the suggestion